guides/guide-for-adding-windows-node.md

```diff
@@ -90,20 +95,88 @@ kube-system kube-flannel-ds-54954 1/1 Running 0 1m
 Now you can add Windows-compatible versions of Flannel and kube-proxy. In order to ensure that you get a compatible version of kube-proxy, you'll need to substitute the tag of the image. The following example shows usage for Kubernetes v1.24.3, but you should adjust the version for your own deployment.
 
 ```bash
-curl -L https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/kube-proxy.yml | sed 's/VERSION/v1.24.3/g' | kubectl apply -f -
-kubectl apply -f https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/flannel-overlay.yml
+# Make sure to modify the version of kube-proxy image to match your kubernetes version
+# before applying it
+wget https://raw.githubusercontent.com/kubernetes-sigs/sig-windows-tools/master/hostprocess/flannel/kube-proxy/kube-proxy.yml
+kubectl apply -f sig-windows-tools/hostprocess/flannel/kube-proxy/kube-proxy.yml
+kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/sig-windows-tools/master/hostprocess/flannel/flanneld/flannel-overlay.yml
 ```
 
->  **Note** If you are using another version of kubernetes on your windows node, change v1.24.3 with your own version .
+>  **Note** If you are using another version of kubernetes on your Windows node, change v1.24.3 with your own version .
 > To find your version of kubernetes run the following command:
 > `kubeadm version`
 
 5. Apply kube-flannel-rbac.yml from sig-windows-tools/kubeadm/flannel
 Next you will need to apply the configuration that allows flannel to spawn pods and keep them running:
 
 ```bash
-git clone https://github.com/kubernetes-sigs/sig-windows-tools
-kubectl apply -f sig-windows-tools/kubeadm/flannel/kube-flannel-rbac.yml
+kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/sig-windows-tools/master/kubeadm/flannel/kube-flannel-rbac.yml
```

Comment:
I think this shouldn't be needed.  Looks like flannel .17 manifests moved all the components to `kube-flannel` namespace.  We should update our manifests to delpoy to `kube-flannel` instead of `kube-system`, which that update we shouldn't need any additional RBAC